### ISSUE TITLE
@ResponseExample support

### ISSUE THREAD

**Issue by vguna** [stoicflame/enunciate#529]
Hi.

Currently I use the auto-generated JSON response examples based on the returned types.
In 95% of the cases this is sufficient. But sometimes it would be great, if one could override the auto-generated ones, and specify custom ones via javadoc. E.g.

``` java
    /**
     * Returns the pet resource instance.
     * 
     * Maybe offer pretty printing? if so, should be an enunciate.xml option - for all examples.
     * @ResponseExample application/json { "id": 4, "name": "mypet" }
     * @ResponseExample application/xml {@code <pet><id>4</id><name>mypet</name></pet>}
     * 
     * OR
     * 
     * This could be hard to parse and to keep formatting.
     * @ResponseExample application/json    {
     *                                          "id": 4,
     *                                          "name": "mypet"
     *                                      }
     * OR
     * 
     * @ResponseExample application/json pet-example.json
     * @ResponseExample application/json relativepathonclasspath/pet-example.json
     * @ResponseExample application/json classpath:pet-example.json
     * @ResponseExample application/json file:///somepath/pet-example.json 
     * 
     * OR
     * 
     * Another idea: for some endpoints it would be enough to just exclude specific properties from auto-generated example (e.g. on PUT).
     * Although some say, one should use different models for this :).
     * @ExcludePropertiesFromResponse id someOtherProperty anotherProperty
     * 
     * @return the pet resource instance.
     */
    @GET
    @Produces({ MediaType.APPLICATION_JSON })
    Pet getPet();
```

WDYT?

**Comment by stoicflame** [comment]
It's a reasonable request.

**Comment by kingferiol** [comment]
This is really interesting. Is there possibility to have something similar for request examples

**Comment by stoicflame** [comment]
This enhancement is currently seeking a sponsor. If anybody is willing to sponsor the work, [reach out to me](http://webcohesion.com/#contact) and I'd be happy to pick it up.

**Comment by stoicflame** [comment]
Thanks to the support of a generous sponsor, this issue has been implemented and will be included in 2.9.

See [Enunciate-Specific Annotations, `@responseExample`](https://github.com/stoicflame/enunciate/wiki/Enunciate-Specific-Annotations#responseexample-javadoc-tag). See also [`@requestExample`](https://github.com/stoicflame/enunciate/wiki/Enunciate-Specific-Annotations#requestexample-javadoc-tag).

**Comment by stoicflame** [comment]
[Enunciate 2.9.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.9.0)
